### PR TITLE
CONSOLE-4118: Switch deprecated select for utilization card filter

### PR DESCRIPTION
**Pull request by sg00dwin**
utilization-card

note: an upstream bug was reported that keyboard selection of an option does not function.
https://github.com/patternfly/patternfly-react/issues/10813

### CODE REVIEW

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx [hunk 1]
```diff
@@ -228,30 +235,46 @@ const UtilizationCardNodeFilter: React.FC<UtilizationCardNodeFilterProps> = ({
     }
     return indexA - indexB;
   });
-  const onToggle = (_event, open: boolean): void => setIsOpen(open);
+  // const onToggle = (_event, open: boolean): void => setIsOpen(open);
```
Comment (rhamilto) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx:238]: Go ahead and delete.

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx [hunk 2]
```diff
@@ -228,30 +235,46 @@ const UtilizationCardNodeFilter: React.FC<UtilizationCardNodeFilterProps> = ({
...
+      role="menu"
       aria-label={t('public~Filter by Node type')}
-      onToggle={onToggle}
       onSelect={onNodeSelect}
-      selections={selectedNodesUpdated}
       isOpen={isOpen}
-      placeholderText={t('public~Filter by Node type')}
-      isPlain
+      selected={selectedNodes}
+      onOpenChange={(onOpenChange) => setIsOpen(onOpenChange)}
```
Comment (rhamilto) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx:273]: Nit: naming.  We're not consistent within the code base, but there is at least one other example that uses `open`.
```suggestion
      onOpenChange={(open) => setIsOpen(open)}
```

**Review by rhamilto — APPROVED** [review]
/lgtm

**Comment by sg00dwin** [comment]
Tech debt so manually adding labels:
/label px-approved
/label docs-approved

QE Approver:
/assign @yapei

**Comment by yanpzhan** [comment]
Launched a cluster against the pr. Following the screenshot in the pr, regression test on Cluster utilization card of Overview page passed.
/label qe-approved

**Pull request merged**